packages/react-app/src/components/QuotaCounter/base.tsx

```diff
@@ -20,7 +20,20 @@ export const BaseQuotaCounter: React.FC<ToolbarState> = ({
   globalTotal,
 }) => (
   <div className={styles.outer_container}>
-    <StatefulTooltip accessibilityType={"tooltip"} content={quotaTooltip}>
+    <StatefulTooltip
+      accessibilityType={"tooltip"}
+      content={quotaTooltip}
+      overrides={{
+        Inner: {
+          style: ({ $theme }) => {
```

Comment:
This can just be
```
{
              color: "var(--backgroundGrey)",
              backgroundColor: "var(--textColorWhite)",
            };
```